src/ext/java/org/opentripplanner/ext/transmodelapi/model/stop/QuayType.java

```diff
@@ -75,7 +86,9 @@ public static GraphQLObjectType create(
           )
           .dataFetcher(environment -> {
             String lang = environment.getArgument("lang");
-            Locale locale = lang != null ? new Locale(lang) : null;
+            Locale locale = lang != null
+              ? GraphQLUtils.getLocale(environment, lang)
```

Comment:
Can you make a reusable method for this, please?

Reply:
Where do you think that method should live?

Reply:
In `GraphQLUtils` or the more Transmodel-specific `GqlUtils`.

Reply:
I'm not sure it's worth it to deduplicate this code, as it only exists for the purpose of supporting the (now) deprecated 'lang' argument, and should be removed when the deprecated arguments are removed.

Given the scenarios:

* 'lang' is not null, we call GraphQLUtils.getLocale(environment, lang) which createas a locale with that value
* 'lang' is null and 'language' is not null, we call GraphQLUtils.getLocale(environment), which extracts the argument value of 'language' and calls getLocale(environment, value) with that value.
* 'lang' is null and 'language' is null, we call GraphQLUtils.getLocale(environment), which tries to extract the arguments, which ends up being null, calling GraphQLUtils.getLocale(environment, null), which in turn tries to find a locale by other means (like headers, or using default language.

Reply:
Since the code is specific to transmodel api, I suppose I could add a method to GqlUtils though